fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -149,6 +156,71 @@ public ParseNode visitShowDatabases(StarRocksParser.ShowDatabasesContext context
         }
     }
 
+    @Override
+    public ParseNode visitCreateMaterializedView(StarRocksParser.CreateMaterializedViewContext context) {
+        boolean ifNotExist = context.IF() != null;
+        String mvName = context.mvName.getText();
+        String comment = context.comment() == null ? null : ((StringLiteral) visit(context.comment().string())).getStringValue();
+        PartitionExpDesc partitionExpDesc = (PartitionExpDesc) visit(context.partitionExpDesc());
+
+        RefreshSchemeDesc refreshSchemeDesc = ((RefreshSchemeDesc) visit(context.refreshSchemeDesc()));
```

Comment:
refreshSchemeDesc may be null ?

Reply:
RefreshDesc can't be null under current conditions, system will throw exceptions.